src/tink/http/Header.hx

```diff
@@ -62,6 +62,7 @@ class Header {
     return byName('content-type').map(ContentType.ofString);
 }
 
+@:forward
 abstract HeaderValue(String) from String to String {
```

Comment:
I'm not so sure `HeaderValue` will always be a `String` underneath. The implicit cast from and to String will no doubt stay, but the value itself may become some parsed representation. I wouldn't want to rush to a decision here. OTOH if you have some useful functions dealing with header values, then adding them to `HeaderValue` directly could benefit everyone ;)